<ui version="4.0" >
 <class>ICQGeneralInfoWidget</class>
 <widget class="QWidget" name="ICQGeneralInfoWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Personal Information</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="2" >
       <widget class="QLabel" name="textLabel10" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>A&amp;ge:</string>
        </property>
        <property name="buddy" >
         <cstring>ageSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="ipEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="maritalLabel" >
        <property name="text" >
         <string>Marital status:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="timezoneEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3" >
       <widget class="QLineEdit" name="marital" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="genderEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="fullNameLabel" >
        <property name="text" >
         <string>&amp;Full name:</string>
        </property>
        <property name="buddy" >
         <cstring>fullNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="birthdayLabel" >
        <property name="text" >
         <string>&amp;Birthday:</string>
        </property>
        <property name="buddy" >
         <cstring>birthday</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="ipLabel" >
        <property name="text" >
         <string>&amp;IP:</string>
        </property>
        <property name="buddy" >
         <cstring>ipEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="fullNameEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="2" column="2" >
       <widget class="QLabel" name="textLabel7" >
        <property name="text" >
         <string>&amp;Timezone:</string>
        </property>
        <property name="buddy" >
         <cstring>timezoneEdit</cstring>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="3" column="1" >
       <widget class="QLineEdit" name="birthday" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="2" column="0" >
       <widget class="QLabel" name="textLabel6" >
        <property name="text" >
         <string>Gen&amp;der:</string>
        </property>
        <property name="buddy" >
         <cstring>genderEdit</cstring>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="3" column="3" >
       <widget class="QSpinBox" name="ageSpinBox" />
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="uinEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="uinLabel" >
        <property name="text" >
         <string>&amp;UIN #:</string>
        </property>
        <property name="buddy" >
         <cstring>uinEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="nickNameEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="nickNameLabel" >
        <property name="text" >
         <string>&amp;Nickname:</string>
        </property>
        <property name="buddy" >
         <cstring>nickNameEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Spoken Languages</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="language1Edit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="language3Edit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="language3Label" >
        <property name="text" >
         <string>Third:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="language2Edit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="language2Label" >
        <property name="text" >
         <string>Second:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="language1Label" >
        <property name="text" >
         <string>First:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Origin</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel2_2" >
        <property name="text" >
         <string>City:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="oStateEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel3_2" >
        <property name="text" >
         <string>Country:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="oCountryEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="oCityEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>466</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>nickNameEdit</tabstop>
  <tabstop>uinEdit</tabstop>
  <tabstop>fullNameEdit</tabstop>
  <tabstop>ipEdit</tabstop>
  <tabstop>genderEdit</tabstop>
  <tabstop>timezoneEdit</tabstop>
  <tabstop>birthday</tabstop>
  <tabstop>ageSpinBox</tabstop>
  <tabstop>marital</tabstop>
  <tabstop>language1Edit</tabstop>
  <tabstop>language2Edit</tabstop>
  <tabstop>language3Edit</tabstop>
  <tabstop>oCityEdit</tabstop>
  <tabstop>oStateEdit</tabstop>
  <tabstop>oCountryEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
